app/views/unattended/report_templates/host_-_products.erb

```diff
@@ -0,0 +1,40 @@
+<%#
+name: Host - Products
+snippet: false
+model: ReportTemplate
+template_inputs:
+- name: Hosts filter
+  required: false
+  input_type: user
+  description: Limit the report only on hosts found by this search query. Keep empty
+    to report on all available hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+require:
+- plugin: katello
+  version: 4.11.0
+-%>
+<%- report_headers 'Host Name', 'Organization', 'Lifecycle Environment', 'Content View', 'Host Collections', 'Virtual', 'Guest of Host', 'OS', 'Arch', 'Sockets', 'RAM', 'Cores', 'SLA', 'Role', 'Usage', 'Products', 'Last Checkin' -%>
+<%- load_hosts(search: input('Hosts filter'), includes: [:operatingsystem, :architecture, :content_view_environments, :organization, :reported_data, :subscription_facet]).each_record do |host| -%>
+<%-   report_row(
+        'Host Name': host.name,
+        'Organization': host.organization,
+        'Lifecycle Environment': host.single_lifecycle_environment,
+        'Content View': host.single_content_view,
+        'Host Collections': host_collections_names(host),
+        'Virtual': host.virtual,
+        'Guest of Host': host.hypervisor_host,
+        'OS': host.operatingsystem,
+        'Arch': host.architecture,
+        'Sockets': host.sockets,
+        'RAM': host.ram,
+        'Cores': host.cores,
+        'SLA': host_sla(host),
+        'Role': host.purpose_role_status_label,
+        'Usage': host.purpose_usage_status_label,
```

Comment:
![image](https://github.com/theforeman/foreman/assets/22042343/6825ea15-a8e8-43d2-8040-f3bf6d97a122)

This is recording the role _status_ and usage _status,_ which tell you if the host's assigned role and usage are "supported by a valid subscription."

Instead these columns should be simply "Role" and "Usage," and should match the values shown for the hosts on the new host details tab Overview tab / System Purpose card. I think the values you want are just `purpose_role` and `purpose_usage`.

Reply:
Ah that makes more sense, alright, I'll update the katello and foreman PRs.